swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id=id, condition=condition)
+        self.fabric = fabric
```

Comment:
We usually want keep our parameters in the same order of parameters used by the parent class
```py
    def __init__(self, id=None, condition=0, fabric="Unknown"):
```

If the parameters were kept in order we could also avoid having to used named parameter syntax on in our `super` call on `line 5`.